website/client/js/controllers/tasksCtrl.js

```diff
@@ -100,8 +99,17 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
       }
     };
 
+    $scope.editTask = function(task, user) {
+      originalTasks.push(_.cloneDeep(task));
+      task._editing = !task._editing;
+      task._tags = !user.preferences.tagsCollapsed;
```

Comment:
Make sure `specHelper.newUser()` sets default values for both of these settings.